apps/aecore/lib/aecore/chain/identifier.ex

```diff
@@ -10,6 +10,7 @@ defmodule Aecore.Chain.Identifier do
 
   alias __MODULE__
   defstruct type: :undefined, value: ""
+  use ExConstructor
```

Comment:
Is this used somewhere?

Reply:
It was added like everywhere. Will remove as we are changing approach.